### ISSUE TITLE
Some documentation suggestions

### ISSUE THREAD

**Issue by chasenlehara** [canjs/can-rest-model#1]
Overall I think the docs are good! Below are some suggestions/questions:

- [x] “restModel mixins-in the…” should probably be “restModel mixes in the…”

- [x] I would expect the `createData: "POST /services/todo"` to be `createData: "POST /services/todos"` (note `todos` instead of `todo`)

- [x] I would reword “…make requests. can-ajax is used by default. But jQuery's…” to be “…make requests; can-ajax is used by default, but jQuery's…”

- [ ] “Returns {connection}: foo” needs to be updated :)

- [x] Some straightforward replacements: “Checkout” => “Check out”, “before hand” => “beforehand”, and “you've got” => “you have”

- [ ] The paragraph under “Nested data type or data types with relationships” has an extra line break

- [x] It seems weird to me that `isSaving` et al. are methods that are observable; why not properties?

**Comment by justinbmeyer** [comment]
@chasenlehara what do you mean by:

> The paragraph under “Nested data type or data types with relationships” has an extra line break

**Comment by justinbmeyer** [comment]
> It seems weird to me that isSaving et al. are methods that are observable; why not properties?

Couldn't mixin properties on instance "late" when DefineMap was first made.  Functions could be easily mixed into the prototype.  So, we mixed in functions.

**Comment by chasenlehara** [comment]
@justinbmeyer Here’s how that paragraph renders at a wider screen width (for me in Chrome):

<img width="942" alt="screen shot 2018-05-07 at 2 47 31 pm" src="https://user-images.githubusercontent.com/10070176/39726949-a46d2618-5205-11e8-8c68-09cc256bea62.png">
